pkg/ops/rawDiskOutput.go

```diff
@@ -88,7 +87,7 @@ func Raw2Azure(source string) (string, error) {
 	}
 	sizeFile := fileInfo.Size()
 
-	if sizeFile%constants.MB != 0 {
+	if int64(size)%constants.MB != 0 {
```

Comment:
size is the old size, but we are writing the header to the file afterwards, so the size has changed. Thats why it updates the stat before checking it, to get the actual real size on disk of it. Not sure if this is ok

Reply:
Azure wants the image size without the "footer" to be a multiple of MB so we don't need to subtract 512 (see the other change). This check becomes less important then, but I changed it anyway to make it work.